shellwhat/State.py

```diff
@@ -1,14 +1,11 @@
 from protowhat.selectors import Dispatcher
 from protowhat.State import State as BaseState
-from shellwhat.parsers import DEFAULT_PARSER, ParseError
+from shellwhat.parsers import DEFAULT_PARSER
 
-class State(BaseState):
 
+class State(BaseState):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    @staticmethod
-    def get_dispatcher():
-        ast_mod = DEFAULT_PARSER(ParseError = ParseError)
-        return Dispatcher(ast_mod.classes, ast_mod)
-        
+    def get_dispatcher(self):
```

Comment:
may be useful to keep staticmethod here, since doesn't use self?

Reply:
The API is shared with other xwhat libraries.

Reply:
AFAIK the only place staticmethod will have an effect is when reading this code, can still be subclassed and overridden etc..

Reply:
I think Python won't make a problem out of it, but IIRC my linter did 😅